dxr/plugins/rust/tests/test_cargo/test_cargo.py

```diff
@@ -0,0 +1,17 @@
+from dxr.testing import DxrInstanceTestCase
+import os
+from os.path import dirname, join
+import subprocess
+import sys
+
+class CargoTests(DxrInstanceTestCase):
+    @classmethod
+    def setup_class(cls):
+        os.environ["RUSTC"] = "./dxr-rustc.sh"
```

Comment:
(Per IRC) we can do this for now, but it would be great if we could get a RUSTC_ARGS into rustc or cargo or a RUST_SAVE_ANALYSIS into rustc at some point.